packages/lib/passesBoundaryCheck/index.ts

```diff
@@ -33,7 +34,7 @@ const isReady = async (source: VectorSource): Promise<boolean> => {
  */
 export const passesBoundaryCheck = async (
   map: Map,
-  boundaryLayerId: string | undefined,
+  boundaryLayerId: LayerBoundPluginOptions['boundaryLayerId'],
```

Comment:
Depending on what is defined in `@polar/lib-custom-types` it may change the behaviour in the future. Would that reason justify a changelog entry in this package or would that be just overkill?

Reply:
For now it's the same, and in the future changes are supposed to trickle through. I don't think this is a change worth announcing, but an internal detail.